app/operate/consensus-validators/consensus-node/page.mdx

```diff
@@ -260,40 +268,41 @@ copied from.
 
 Run the following command to quick-sync from a snapshot:
 
-
-**Mainnet Beta:**
-```bash
-cd $HOME
-rm -rf ~/.celestia-app/data
-mkdir -p ~/.celestia-app/data
-SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
-    egrep -o ">{{constants['mainnetChainId']}}.*tar" | tr -d ">")
-aria2c -x 16 -s 16 -o celestia-snap.tar "https://snaps.qubelabs.io/celestia/${SNAP_NAME}"
-tar xf celestia-snap.tar -C ~/.celestia-app/data/
-```
-
-**Mocha:**
-```bash
-cd $HOME
-rm -rf ~/.celestia-app/data
-mkdir -p ~/.celestia-app/data
-SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
-    egrep -o ">{{constants['mochaChainId']}}.*tar" | tr -d ">")
-aria2c -x 16 -s 16 -o celestia-snap.tar "https://snaps.qubelabs.io/celestia/${SNAP_NAME}"
-tar xf celestia-snap.tar -C ~/.celestia-app/data/
-```
-
-**Arabica:**
-```bash
-cd $HOME
-rm -rf ~/.celestia-app/data
-mkdir -p ~/.celestia-app/data
-SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
-    egrep -o ">{{constants['arabicaChainId']}}.*tar" | tr -d ">")
-aria2c -x 16 -s 16 -o celestia-snap.tar "https://snaps.qubelabs.io/celestia/${SNAP_NAME}"
-tar xf celestia-snap.tar -C ~/.celestia-app/data/
-```
-
+<Tabs items={['Mainnet Beta', 'Mocha', 'Arabica']}>
+  <Tabs.Tab>
+    ```bash
+    cd $HOME
+    rm -rf ~/.celestia-app/data
+    mkdir -p ~/.celestia-app/data
+    SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
+        egrep -o ">{{constants['mainnetChainId']}}.*tar" | tr -d ">")
+    aria2c -x 16 -s 16 -o celestia-snap.tar "https://snaps.qubelabs.io/celestia/${SNAP_NAME}"
+    tar xf celestia-snap.tar -C ~/.celestia-app/data/
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash
+    cd $HOME
+    rm -rf ~/.celestia-app/data
+    mkdir -p ~/.celestia-app/data
+    SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
+        egrep -o ">{{constants['mochaChainId']}}.*tar" | tr -d ">")
+    aria2c -x 16 -s 16 -o celestia-snap.tar "https://snaps.qubelabs.io/celestia/${SNAP_NAME}"
+    tar xf celestia-snap.tar -C ~/.celestia-app/data/
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash
+    cd $HOME
+    rm -rf ~/.celestia-app/data
+    mkdir -p ~/.celestia-app/data
+    SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
+        egrep -o ">{{constants['arabicaChainId']}}.*tar" | tr -d ">")
+    aria2c -x 16 -s 16 -o celestia-snap.tar "https://snaps.qubelabs.io/celestia/${SNAP_NAME}"
+    tar xf celestia-snap.tar -C ~/.celestia-app/data/
+    ```
+  </Tabs.Tab>
+</Tabs>
 
 > **Tip:**
```

Comment:
This tip is coming back "empty", but that's just how it's being rendered. We can address by looking for a "tip" component.